### PR TITLE
Ubuntu/jammy: Add Depends: python3-debconf

### PR DESCRIPTION
**Pull request by aciba90**
Depends: python3-debconf added in https://github.com/canonical/cloud-init/pull/1506 and https://github.com/canonical/cloud-init/pull/1472

SC-1102

(Do not squash)

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,10 @@
+cloud-init (22.2-0ubuntu1~22.04.1) UNRELEASED; urgency=medium
```
Comment (blackboxsw) [inline debian/changelog:1]: Note we can't reuse an already released version 22.2-0ubuntu1~22.04.1 in debian/changelog, so we'd want this to be
```suggestion
cloud-init (22.2-0ubuntu1~22.04.2) UNRELEASED; urgency=medium
```

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+
+  * d/control:
```
Comment (blackboxsw) [inline debian/changelog:3]: Let's make this just one line

File: debian/control [hunk 3]
```diff
@@ -38,6 +38,7 @@ Depends: cloud-guest-utils | cloud-utils,
          netplan.io,
          procps,
          python3,
+         python3-debconf,
```
Comment (blackboxsw) [inline debian/control:41]: We also need this in Build-Depends too. See https://github.com/canonical/cloud-init/pull/1511 for context.

**Review by blackboxsw — CHANGES_REQUESTED** [review]

**Comment by aciba90** [comment]
Thanks for the review @blackboxsw. I have applied the requested changes. Ready to be reviewed.

**Review by blackboxsw — APPROVED** [review]

**Pull request merged**